c2corg_ui/static/js/card.js

```diff
@@ -107,4 +107,94 @@ app.CardController.prototype.createURL = function() {
 };
 
 
+/**
+ * Gets the global ratings for each activity of a route.
+ * @export
+ * @return {Object} rating
+ */
+app.CardController.prototype.getGlobalRatings = function() {
+  var doc = this.doc;
+  var rating = {};
+
+  for (var i = 0; i < doc.activities.length; i++) {
+    var a = doc.activities[i];
+    if (a === 'rock_climbing' || a === 'mountain_climbing' || a === 'snow_ice_mixed') {
+      rating['global_rating'] = doc.global_rating;
+    } else if (a === 'via_ferrata') {
+      rating['via_ferrata_rating'] = doc.via_ferrata_rating;
+    } else if (a === 'snowshoeing') {
+      rating['snowshoe_rating'] = doc.snowshoe_rating;
+    } else if (a === 'hiking') {
+      rating['hiking_rating'] = doc.hiking_rating;
+    } else if (a === 'skitouring') {
+      rating['labande_global_rating'] = '';
+      if (doc.labande_global_rating && doc.labande_labande_global_rating) {
+        rating['labande_global_rating'] = doc.labande_global_rating + '/' + doc.labande_labande_global_rating;
```

Comment:
I guess you mean ``ski_rating/ski_exposition_rating`` like in https://github.com/c2corg/v6_ui/issues/398#issuecomment-234203552